src/find_ice_labels.jl

```diff
@@ -1,37 +1,163 @@
-# Type of ref_image_band_2
-__T__ = SubArray{
-    Float64,
-    2,
-    Base.ReinterpretArray{Float64,3,RGB{Float64},Matrix{RGB{Float64}},true},
-    Tuple{Int64,Base.Slice{Base.OneTo{Int64}},Base.Slice{Base.OneTo{Int64}}},
-    false,
-}
+using ImageBinarization: AbstractImageBinarizationAlgorithm, binarize
+"""
+    IceDetectionAlgorithm
+
+Functors to detect ice regions in an image.
+
+Each algorithm `a` with parameters `kwargs...` can be called like:
+- `binarize(image, a(; kwargs...))` 
+- `find_ice(image, a(; kwargs...))`
```

Comment:
The alias `find_ice` may be confusing if it only is for the binarization algorithms. Are you thinking of this as a function that could accept all the IFT Ice Detection Algorithms (even later ones that aren't binary?)

Reply:
Honestly, I'm happy to get rid of it. It's just an alias for `binarize` after all. We can always add a different alias else back in later, if we need it. The assumption that the results will be binary are pretty deep in the code, so we'd need to do a lot more work to support a general non-binary algorithm.